test/gateways/trexle_test.exs

```diff
@@ -54,6 +56,275 @@ defmodule Gringotts.Gateways.TrexleTest do
     description: "For our valued customer, Mr. Potter"
   ]
 
+  @invalid_amount_response ~s/{"error":"Payment failed","detail":"Amount must be at least 50 cents"}/
+  @invalid_card_response ~s/{"error":"Payment failed","detail":"Your card's expiration year is invalid."}/
+  @valid_request_response ~s/{"response":{"token":"charge_3e89c6f073606ac1efe62e76e22dd7885441dc72","success":true,"captured":false}}/
+  @invalid_token_response ~s/{"error":"Capture failed","detail":"invalid token"}/
+  @valid_token_response ~s/{"response":{"token":"#{@valid_token}","success":true,"captured":true,"amount":299,"status_message":"Transaction approved"}}/
+  @invalid_token_response_for_refund ~s/{"error":"Refund failed","detail":"invalid token"}/
+
+  describe "Store integration point testing" do
```

Comment:
We'll just call it `describe "store"`, and then the `test` names complete the sentence like:
"`store` with valid card"
"`refund` with invalid token"
"`refund` with valid charge token"
...

Reply:
Yes, Agreed. Since we are going to remove mock

Reply:
Done